srcpkgs/duplicity/template

```diff
@@ -12,19 +12,13 @@ license="GPL-2.0-or-later"
 homepage="http://duplicity.nongnu.org/"
 changelog="http://duplicity.nongnu.org/vers8/CHANGELOG"
 distfiles="http://launchpad.net/duplicity/${version%.*}-series/${version}/+download/duplicity-${version}.tar.gz"
-checksum=2cfcd212834979751310d9df1a954b949f86b084efd5ad1f97dd744780ef9d1b
+checksum=a273b1e74d8d0b7825572f51be9c8284e579961e01f4fc1473048e95dce49984
 
 post_extract() {
 	chmod 0644 bin/*.1
 }
 
-do_check() {
```

Comment:
Did upstream completely remove their tests? Right now there isn't a `test` target in `setup.py`, but they could be using `pytest` directly.

Reply:
Looks like they did change it to pytest. will update